ouroboros-network/src/Ouroboros/Network/NodeToClient/Version.hs

```diff
@@ -35,7 +35,7 @@ data NodeToClientVersion
     | NodeToClientV_13
     -- ^ enabled @CardanoNodeToClientVersion9@, i.e., Babbage
     | NodeToClientV_14
-    -- ^ added @GetPoolState, @GetSnapshots
+    -- ^ added @GetPoolDistr, @GetPoolState, @GetSnapshots
```

Comment:
:+1: `NodeToClientV_14` hasn't been yet released. `node-1.35.x` comes with `NodeToClientV_13` as the latest version.